pyvista/errors.py

```diff
@@ -4,6 +4,10 @@
 class MissingDataError(ValueError):
     """Exception when data is missing, e.g. no active scalars can be set."""
 
+    def __init__(self, message='No data available.'):
+        """Call the base class constructor with the custom message."""
+        super().__init__(message)
```

Comment:
Do we need this default? If yes, should we use it elsewhere? And should we also upgrade the `AmbiguousDataError` the same way?

Reply:
Added in 45b20759a970a65f1e6d503156f03b9ddf5a91f2